lib/drivers/bayerContourNext.js

```diff
@@ -161,17 +167,33 @@ module.exports = function (config) {
       var pInfo = pString[4].split('^');
       var sNum = pInfo[2].match(/^\d+\-\s*(\w+)/);
       var records = data;
-      var threshold = pString[5].split('^')[7].match(/^.+\=(\d{2})(\d{3})/);
+      var threshold = null;
+      var thrs = pString[5].split('^');
+
+      for (var i = 0; i < thrs.length; i++){
+        var val = thrs[i].match(/^(\w+)\=/);
+        if (val[1] === 'V'){
+          threshold = thrs[i].match(/^.+\=(\d{2})(\d{3})/);
+          break;
+        }
+      }
 
       var devInfo = {
         model: pInfo[0],
         serialNumber: sNum[1],
         nrecs: pString[6],
         rawrecords: records,
-        lowT: parseInt(threshold[1]),
-        hiT: parseInt(threshold[2])
       };
 
+      if(threshold){
+        devInfo.lowT = parseInt(threshold[1]);
+        devInfo.hiT = parseInt(threshold[2]);
+      }else{
+        devInfo.uThres = true;
```

Comment:
I'd prefer a slightly more explicit variable name here - the full `unreportedThres` maybe? I remembered this from our conversation but coming new to the code I probably wouldn't know what it meant without looking for it elsewhere.